arm/Microsoft.AnalysisServices/servers/deploy.bicep

```diff
@@ -40,8 +41,8 @@ param diagnosticEventHubAuthorizationRuleId string = ''
 param diagnosticEventHubName string = ''
 
 @allowed([
-  'CanNotDelete'
   'NotSpecified'
+  'CanNotDelete'
```

Comment:
Was this changed to trigger the pipeline and publishing? If so it can be reverted back

Reply:
Fixed